src/data-services/ce-backend.js

```diff
@@ -59,6 +59,42 @@ function getData(
   });
 }
 
+// TODO: https://github.com/pacificclimate/climate-explorer-frontend/issues/124
+function getStats (
+    { ensemble_name, model_id, variable_id, experiment, area }, timeidx
+) {
+  // Query the "multistats" API endpoint.
+  // Gets an object from each qualifying dataset file with the following
+  // information:
+  //  {
+  //   unique_ID: {
+  //      min
+  //      max
+  //      mean
+  //      median
+  //      stdev
+  //      ncells
+  //      units
+  //      time (median time represented by the dataset)
+  //      modtime (last time dataset was modified)
+  //    }
+  //  }
+  let queryExpString = guessExperimentFormatFromVariable(variable_id, experiment);
```

Comment:
`const`
Also, wouldn't this be more naturally named `experiment` or `emission`?